.github/workflows/rust.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+
+on: [push]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
This matrix is just a waste of build time. We use only `stable` versions, no need to check other cases